.github/workflows/kind.yaml

```diff
@@ -26,11 +26,12 @@ jobs:
           make build
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
-          timeout 1m make deploy IMG=flotta-operator || retVal=$?
+          make deploy IMG=flotta-operator TARGET=kind || retVal=$?
```

Comment:
pls change the order to:
`IMG=flotta-operator TARGET=kind make deploy`

Reply:
done